src/apps/board/BoardApp.js

```diff
@@ -141,11 +184,47 @@
                 boardConfig.enableCrossColumnRanking = false;
                 boardConfig.cardConfig.showRankMenuItems = false;
             }
+
+            if (this.getSetting('eventType')=="Send") {
+                Ext.merge(boardConfig, {
+                    cardConfig: {
+                        xtype: 'myappcard',
+                        listeners: {
+                            'cardhover': {
+                                 fn: function(a,b,c) {
+                                    console.log("sending click:",b);
+                                    that.publish("cardClick",b);
+                                 },   
+                                 scope: this
+                             }
+                        }
+                    }
+                } )
+            }
+
             return boardConfig;
         },
 
         getSettingsFields: function() {
-            return Rally.apps.board.Settings.getFields(this.getContext());
+
+            var eventsStore = new Ext.data.ArrayStore({
```

Comment:
you could also just directly add this to the getFields method in the Settings class...